nwb_conversion_tools/nwbconverter.py

```diff
@@ -62,21 +80,23 @@ def get_metadata(self):
             NWBFile=dict(
                 session_description="no description",
                 identifier=str(uuid.uuid4()),
-                session_start_time=datetime.strptime('1900-01-01 00:00:00', '%Y-%m-%d %H:%M:%S'),
+                session_start_time=datetime(1900, 1, 1)
             )
         )
         for interface in self.data_interface_objects.values():
             interface_metadata = interface.get_metadata()
             metadata = dict_deep_update(metadata, interface_metadata)
         return metadata
 
-    def run_conversion(self, metadata, nwbfile_path=None, save_to_file=True, conversion_options=None):
+    def run_conversion(self, metadata: dict, nwbfile_path=None, save_to_file: bool = True,
+                       conversion_options: dict = None):
         """Build nwbfile object, auto-populate with minimal values if missing."""
         if conversion_options is None:
             conversion_options = dict()
-        nwbfile_kwargs = dict()
-        if 'NWBFile' in metadata:
-            nwbfile_kwargs.update(metadata['NWBFile'])
+        else:
+            validate(instance=conversion_options, schema=self.get_conversion_options_schema())
```

Comment:
This would be problematic in this specific situation: The conversion_options_schema has required properties, but when calling run_conversion() the users does not pass them

Reply:
I don't think any conversion options should be required